supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesBackendFileHashing.scala

```diff
@@ -0,0 +1,21 @@
+package cromwell.backend.impl.tes
+
+import akka.event.LoggingAdapter
+import cromwell.backend.callcaching.FileHashingActor.SingleFileHashRequest
+import cromwell.core.path.DefaultPathBuilder
+import cromwell.util.TryWithResource._
+
+import scala.language.postfixOps
+import scala.util.Try
+
+private[tes] object TesBackendFileHashing {
+  def getMd5Result(request: SingleFileHashRequest, log: LoggingAdapter): Try[String] = {
+    val path = DefaultPathBuilder.build(request.file.valueString) recover {
+      case failure => throw new RuntimeException("Failed to construct path to hash", failure)
+    } get
+
+    tryWithResource(() => path.newInputStream) { inputStream =>
+      org.apache.commons.codec.digest.DigestUtils.md5Hex(inputStream)
```

Comment:
ToL: Note for the future, this is fine for passing centaur tests, but takes too long on real BAM files. Hopefully whenever #1843 is implemented it will be reusable by this backend also.